opsdroid/events.py

```diff
@@ -0,0 +1,169 @@
+"""Classes to describe different kinds of possible event."""
+
+from abc import ABC
```

Comment:
Can we sort these import nicely? I'm surprise the linter didn't pick it up.

E.g

```
import asyncio
from abc import ABC
from copy import copy
from datetime import datetime
from random import randrange

from opsdroid.helper import get_opsdroid
```